Create ui-c8y-1023-57-0-homepage-update-11299-graft-release-cd.md (#4397)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1023-57-0-homepage-update-11299-graft-release-cd.md

---
date: ""

> title: Device Management Homepage Refresh

Reviewer: ```suggestion
title: Device Management application homepage refresh
```

> title: homepage update (#11299) [GRAFT][release/cd]

Reviewer 2: ```suggestion
title: Device Management Homepage Refresh
```

product_area: Device management & connectivity
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-5259
version: 1023.57.0
---

> homepage update (#11299) [GRAFT][release/cd]

Reviewer 2: ```suggestion
The Device Management homepage has been refreshed by removing the standard alarm and quick-link widgets to make room for a new Welcome Widget. This update provides a more intuitive entry point for users and includes a direct trigger for the thin-edge.io wizard.
```

> The Device Management homepage has been refreshed by removing the standard alarm and quick-link widgets to make room for a new Welcome Widget. This update provides a more intuitive entry point for users and includes a direct trigger for the thin-edge.io wizard.

Reviewer: ```suggestion
The Device Management application homepage has been refreshed by removing the standard "Alarm" and "Quick links" widgets to make room for a new welcome widget. This update provides a more intuitive entry point for users and includes a direct link to the thin-edge.io setup wizard.
```
